stubs/openpyxl/openpyxl/worksheet/worksheet.pyi

```diff
@@ -51,6 +51,7 @@ class Worksheet(_WorkbookChild):
     ORIENTATION_PORTRAIT: Final = "portrait"
     ORIENTATION_LANDSCAPE: Final = "landscape"
 
+    _cells: dict[tuple[int, int], _CellOrMergedCell] # private but very useful to understand typing
```

Comment:
I could just comment this out before merging, Users shouldn't be using this, but it's a useful note.